platform/ui/src/components/content/viewportDownloadForm/ViewportDownloadForm.js

```diff
@@ -204,19 +263,33 @@ const ViewportDownloadForm = ({
               min={minimumSize}
               max={maximumSize}
               value={width}
-              label={t('Image width (px)')}
+              label={t('imageWidth')}
               onChange={onWidthChange}
             />
+            {renderErrorHandler('width')}
```

Comment:
This is cool beans for now. It might be worth creating a follow-up issue for form inputs that have a standard way of showing validation errors.